crates/apub/src/objects/post.rs

```diff
@@ -228,8 +223,6 @@ impl Object for ApubPost {
 
     let alt_text = first_attachment.cloned().and_then(Attachment::alt_text);
 
-    let url = proxy_image_link_opt_apub(url, context).await?;
```

Comment:
Culprit was here, these urls are no longer proxied, but also it was proxying even for non-image posts.